### DISCUSSION TITLE
HELP please, unable to upgrade. open-webui siliently stops after v0.3.21

### DISCUSSION THREAD

**Discussion by Rathna-K** [open-webui/open-webui#6801] [General]
I am using a basic docker command inside an Atom processor x86/ x64 PC running Debian v11 Bullseye. Docker version is 27.0.3. 
When I run this command, any other version higher than 21 breaks and container keeps restarting. Doesn't happen in v21 or lower. Something broken after this version between the OS, Docker and this but I cannot find anything in the debug log as it just quits.  please help!! 

`
docker run -p 3080:8080 -e OLLAMA_BASE_URL=http://192.168.1.1:11434 -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:dev
`
here is the log:
`

/app/backend/open_webui
/app/backend
/app
INFO:open_webui.env:GLOBAL_LOG_LEVEL: DEBUG
INFO:open_webui.env:AUDIO_LOG_LEVEL: DEBUG
INFO:open_webui.env:COMFYUI_LOG_LEVEL: DEBUG
INFO:open_webui.env:CONFIG_LOG_LEVEL: DEBUG
INFO:open_webui.env:DB_LOG_LEVEL: DEBUG
INFO:open_webui.env:IMAGES_LOG_LEVEL: DEBUG
INFO:open_webui.env:MAIN_LOG_LEVEL: DEBUG
INFO:open_webui.env:MODELS_LOG_LEVEL: DEBUG
INFO:open_webui.env:OLLAMA_LOG_LEVEL: DEBUG
INFO:open_webui.env:OPENAI_LOG_LEVEL: DEBUG
INFO:open_webui.env:RAG_LOG_LEVEL: DEBUG
INFO:open_webui.env:WEBHOOK_LOG_LEVEL: DEBUG
INFO:open_webui.apps.webui.internal.wrappers:Connected to SQLite database
INFO:open_webui.apps.webui.internal.db:Starting migrations
DEBUG:peewee:('CREATE TABLE IF NOT EXISTS "migratehistory" ("id" INTEGER NOT NULL PRIMARY KEY, "name" VARCHAR(255) NOT NULL, "migrated_at" DATETIME NOT NULL)', [])
DEBUG:peewee:('SELECT "t1"."id", "t1"."name", "t1"."migrated_at" FROM "migratehistory" AS "t1" ORDER BY "t1"."id"', [])
INFO:open_webui.apps.webui.internal.db:There is nothing to migrate
Running migrations
DEBUG:passlib.utils.compat:loaded lazy attr 'SafeConfigParser': <class 'configparser.ConfigParser'>
DEBUG:passlib.utils.compat:loaded lazy attr 'NativeStringIO': <class '_io.StringIO'>
DEBUG:passlib.utils.compat:loaded lazy attr 'BytesIO': <class '_io.BytesIO'>
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
INFO  [open_webui.env] 'DEFAULT_LOCALE' loaded from the latest database entry
INFO  [open_webui.env] 'DEFAULT_PROMPT_SUGGESTIONS' loaded from the latest database entry
WARNI [open_webui.env]

WARNING: CORS_ALLOW_ORIGIN IS SET TO '*' - NOT RECOMMENDED FOR PRODUCTION DEPLOYMENTS.

INFO  [open_webui.env] Embedding model set: sentence-transformers/all-MiniLM-L6-v2
INFO  [open_webui.apps.audio.main] whisper_device_type: cpu
`

**Comment by werner1234ik** [comment]
What is the installed version of Docker for OpenWebUI? Is it also a development version?
ghcr.io/open-webui/open-webui:dev
